sys/cbor/cbor.c

```diff
@@ -522,7 +522,7 @@ size_t cbor_deserialize_float(const cbor_stream_t *stream, size_t offset, float
 
     if (*data == CBOR_FLOAT32) {
         *val = ntohf(*(uint32_t *)(data + 1));
-        return 4;
+        return 5;
```

Comment:
It includes also the flags field (as #5420 points out). If the 4 would be correct the 9 below in ll. 549-550 would be incorrect.